fbgemm_gpu/bench/quantize_ops_benchmark.py

```diff
@@ -44,6 +45,9 @@ def cli() -> None:
     num_rows=st.sampled_from([2**n for n in range(4, 10)]),
 )
 def bench(
+=======
+def bench_impl(
+>>>>>>> 33c5e06... update
```

Comment:
I don't think this PR is ready to review. Can you please check the whole change again?

Reply:
Quite sorry for this, I overlooked the conflict. It's resolved.

Reply:
Thanks for your quick fix!